### PR TITLE
Giphy API (specific) and CFHour monitoring for waffle enjoyment

### PR DESCRIPTION
**Pull request by coldfumonkeh**
Created a new giphy plugin to target the Giphy API specifically.

Also added monkehTweets to the 'standard' docs because, you know, self-promotion FTW.

CFHour feed relay (cfbugs but for waffle) added to get latest show notifications and link to show mp3.

### CODE REVIEW

File: plugins/cfdocs.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ module.exports = (function(){
             cfclient: '<cfclient></cfclient> → returns a pink slip, because if you use this shit you should be fired. ~ http://www.codecademy.com/en/tracks/javascript',
             cf_socialplugin: '<cf_socialplugin .. /> → returns a bunch of outdated junk that would have been better as a community project dear god what have we done... we should have just given them a package manager like they\'ve been requesting for years ~ http://cfdocs.org/cf_socialplugin',
             cfscriptref: 'https://github.com/daccfml/cfscript/blob/master/cfscript.md',
-            cfdownloads: 'http://www.gpickin.com/cfrepo/'
+            cfdownloads: 'http://www.gpickin.com/cfrepo/',
+						monkehtweets: 'https://github.com/coldfumonkeh/monkehTweets'
```
Comment (atuttle) [inline plugins/cfdocs.js]: Whitespace differences?!

**Comment by atuttle** [comment]
Does the giphy api filter out NSFW gifs by default? If not, and they offer a flag to do it, we'll definitely want to enable it.

**Comment by coldfumonkeh** [comment]
Rating option enabled (PG). This strips out PG-13 and R-rated gifs but still leaves a little over the Y and G ratings.

http://media.giphy.com/media/BUlwrDd97bCYE/giphy.gif

File: plugins/cfdocs.js (hunk 1 above)
Comment (atuttle) [inline plugins/cfdocs.js]: _ahem_

**Comment by coldfumonkeh** [comment]
Need anything else? :)

**Comment by coldfumonkeh** [comment]
image

**Comment by coldfumonkeh** [comment]
huh?

**Comment by atuttle** [comment]
You have some really insane indenting there... 2 tabs?

**Comment by atuttle** [comment]
For example, look around here: https://github.com/coldfumonkeh/zoidbox/commit/5e8269eb80b858955a46be0c00cf548efa956626#diff-d85dc630e48d8649efb7b5a1cb89a1faL117

**Pull request merged**